### ISSUE TITLE
Problem with p:set-attributes-002 (NW)

### ISSUE THREAD

**Issue by xml-project** [xproc/3.0-test-suite#694]
I think this test needs to be changed:
`attributes="map { Q{http://www.w3.org/2000/xmlns/}x: '5' }"`

is not a valid XPath expression. [XPath 3.1.](https://www.w3.org/TR/xpath-31/#doc-xpath31-BracedURILiteral) says:
````
It is a [static error](https://www.w3.org/TR/xpath-31/#dt-static-error) [[err:XQST0070](https://www.w3.org/TR/xpath-31/#ERRXQST0070)] if the namespace URI for an EQName is http://www.w3.org/2000/xmlns/.
````
I would suggest:

1. Allowing XS0107 (error in XPath expression) as a possible result
2. Rewriting the test without BracedURILitera.l

**Comment by ndw** [comment]
Yeah, I struggled with that too. I spent five minutes trying to rework the expression to avoid the braced literal without success. If it's obvious to you how to do that, please go for it. Otherwise XS0107 is fine too. (In fact, I had to hack things a bit to avoid that error, but I still don't *really* like how I made it work.)

**Comment by xml-project** [comment]
Closed with pr #695
